images/origin/install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
These need to be structured, commented, and follow our guidelines for bash (all variables must be within double quotes, etc).

Will these commands only run on redhat distros?  If so, add a comment at the top of this file describing.